sections/platform-section/platform-hero.js

```diff
@@ -28,7 +26,6 @@ const PlatformHeroHeading = styled(Heading)`
 
   @media only screen and (min-width: ${TABLET_SMALL_SIZE}) {
     font-size: ${rem('48px')};
```

Comment:
For some reason, this is being converted to 2.25rem = 36px

Reply:
ok, I'll add another level of specificity